### ISSUE TITLE
Turkey

### ISSUE THREAD

**Issue by oliverjam** [yvonne-liu/friendsgiving#5]
### Ingredients
- [ ] Turkey (breast/thigh?)

**Comment by yvonne-liu** [comment]
> Turkey (breast/thigh?)

What no. Whole and stuffed.

**Comment by yvonne-liu** [comment]
> Turkey (breast/thigh?)

What no. Whole and stuffed.

**Comment by yvonne-liu** [comment]
Okay, I'll stop GitHub-ing on my phone.

**Comment by finnhodgkin** [comment]
Whole and stuffed is a lot of turkey for few meat people.

**Comment by starsuit** [comment]
I propose this:
https://www.chefsteps.com/activities/turkey-roulade

**Comment by oliverjam** [comment]
Yeah we are not doing an entire turkey lol. Sam and I have both made that roulade before for Christmas and it works pretty amazingly

**Comment by oliverjam** [comment]
So the decision may have been made for us—I asked my Mum to look out for whole turkey thighs in the local Sainsbury's because they don't always have them, but she went ahead and just bought us two 😄
